### PR TITLE
Add Velero module

### PR DESCRIPTION
**Pull request by smtm001**

### CODE REVIEW

File: modules/velero/variables.tf [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  default = {
+    CriticalAddonsOnly = "true"
+  }
+}
+
+variable "critical_addons_node_tolerations" {
+  description = "Config for node tolerations for workloads"
+  type        = list(any)
+  default = [{
+    key      = "CriticalAddonsOnly"
```
Comment (v47) [inline modules/velero/variables.tf]: I think the indentation is wrong.

File: modules/velero/main.tf [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    name: velero
+    annotations:
+      eks.amazonaws.com/role-arn: ${aws_iam_role.velero.arn}
+
+# Resources to Velero deployment
+resources:
+  requests:
+    cpu: 100m
+    memory: 128Mi
+  limits:
```
Comment (v47) [inline modules/velero/main.tf]: This should be configurable. In case of large clusters, it will die.

**Review by v47 — APPROVED** [review]
Can you plz add the description.

**Comment by v47** [comment]
@tvc123 can you take a look?

**Review by d3vb0ox — APPROVED** [review]

**Pull request merged**